modules/getting-started/p_login-admin.adoc

```diff
@@ -0,0 +1,24 @@
+// Metadata created by nebel
+//
+// UserStory: As an RH SSO customer, I want to login into my account
+
+[id="login-admin"]
+= Logging into the Admin Console
+
+After you create the initial admin account, you can log in to the Admin Console.
+
+.Procedure
+. Click the *Administration Console* link on the *Welcome* page or go directly to http://localhost:8080/auth/admin/ (the console URL). 
+
++
+[NOTE]
+====
+The Administration Console is generally referred to as the Admin Console for short in {project_name} documentation.
```

Comment:
In the first instance in the doc, this is referred to as "Keycloak Admin Console" - might need to check the consistency of this throughout the doc. Maybe use an attribute instead as for {project_name}?

Reply:
Also noticed "Keycloak Admin Console" at the start of Chapter 2

Reply:
Thanks, Ashleigh. Actually, I think I only use Keycloak for sections that are part of a conditional content, so they don't show up in the SSO version.  I hope that is what you mean.